tests/fastai/test_fastai_autolog.py

```diff
@@ -36,25 +47,39 @@ def iris_data():
     iris = datasets.load_iris()
     X = pd.DataFrame(iris.data[:, :2], columns=iris.feature_names[:2])
     y = pd.Series(iris.target, name="label")
-    return (
-        TabularList.from_df(pd.concat([X, y], axis=1), cont_names=list(X.columns))
-        .split_by_rand_pct(valid_pct=0.1, seed=42)
-        .label_from_df(cols="label")
-        .databunch()
+    return TabularDataLoaders.from_df(
+        df=pd.concat([X, y], axis=1), cont_names=list(X.columns), y_names="label"
     )
 
 
-def fastai_model(data, **kwargs):
+def fastai_tabular_model(data, **kwargs):
     return tabular_learner(data, metrics=accuracy, layers=[5, 3, 2], **kwargs)
 
 
+def mnist_path():
+    mnist = untar_data(URLs.MNIST_TINY)
+    return mnist
+
+
+@pytest.fixture(scope="session")
+def mnist_data():
+    mnist = untar_data(URLs.MNIST_TINY)
+    return ImageDataLoaders.from_folder(mnist, num_workers=0)
+
+
+def fastai_visual_model(data, **kwargs):
+    return cnn_learner(data, models.resnet18, normalize=False, **kwargs)
```

Comment:
I think training a ResNet model with the MNIST dataset takes a long time. Can we use a smaller dataset and a simpler model?

Reply:
In this run, `pytest tests/fastai/test_fastai_autolog.py --large` takes about 10 minutes:
https://github.com/mlflow/mlflow/pull/4715/checks?check_run_id=3583140216

Reply:
I didn't check every model there, but resnet18 has "only" 11M params. It appears to be the smallest model among those offered: https://fastai1.fast.ai/vision.models.html

Reply:
@jinzhang21  
I found a way to create a fine-tunable tabular model. Can I push some commits?

Reply:
@jinzhang21 Puhsed a commit to update the code. Now it takes about 3 min. to run tests for fastai autologging: https://github.com/mlflow/mlflow/pull/4715/checks?check_run_id=3605777911

Reply:
Super nice! Thanks for the change, Haru!